origin_ca.go

```diff
@@ -0,0 +1,147 @@
+package cloudflare
+
+import (
+	"encoding/json"
+
+	"github.com/pkg/errors"
+)
+
+// OriginCA resource
+// This is the origin_ca resource definition (as defined here:
+// https://api.cloudflare.com/#origin-ca || https://api.cloudflare.com/#cloudflare-ca
```

Comment:
Actually, it seems like all the references pointing at `#origin-ca...` has been changed to `#cloudflare-ca...` in the documentation.